README.md

```diff
@@ -241,6 +242,10 @@
 
 * [MyNixOS](https://mynixos.com/) - Graphical editor for Nix flakes. Create and manage configurations and modules for NixOS and Nix home-manager. Rather a Nix generator than a Nix editor, because it does not allow to import Nix files.
 
+## NixOS Base Configurations
```

Comment:
Maybe the section should be named `Configuration templates` or something.

Reply:
Sure! I'll do that